validator/testdata/feature_tests/amp-audioburst-audio.out

```diff
@@ -0,0 +1 @@
+PASS
```

Comment:
Optional: Would you like to implement a negative case?

Reply:
Hi, thank you for your review. I'm sorry, but i'm not sure that I understood what do you mean on negative case? Where to implement it?